common/scala/src/main/scala/whisk/http/ErrorResponse.scala

```diff
@@ -46,7 +47,10 @@ object Messages {
     val conformanceMessage = "Resource by this name exists but is not in this collection."
     val corruptedEntity = "Resource is corrupted and cannot be read."
 
-    val systemOverloaded = "System is overloaded, try again later."
+    /**
+     * Standard message for reporting deprecated runtimes.
+     */
+    def runtimeDeprecated(e: Exec) = s"The '${e.kind}' runtime is no longer supported. You may GET or DELETE but not PUT or POST."
```

Comment:
Maybe use more human terminology: You may read and delete but not update or invoke your action.